pom.xml

```
@@ -8,7 +8,7 @@
   </parent>
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.332.4</jenkins.version>
+    <jenkins.version>2.346.1</jenkins.version>
```

Let's go all the way to 2.346.3 as recommended in the ["Choosing a version" page](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/)
```suggestion
    <jenkins.version>2.346.3</jenkins.version>
```